examples/sample-project/build.gradle

```diff
@@ -58,11 +68,157 @@ task loadZipCodeBoundaryExampleData(type: com.marklogic.gradle.task.MlcpTask) {
   port = mlAppConfig.restPort
   database = mlAppConfig.contentDatabaseName
   input_file_path = "data/zipcodes"
-  transform_module = "/transform/wrap-geojson.sjs"
   input_compressed = "true"
-  batch_size = 1
-  transaction_size = 1
   output_collections = "zipcodes,test-data"
```

Comment:
Shouldn't we change "test-data" to "example-data"?

Reply:
Agree, collection name changed.